Rock.Mailgun/app.config

```diff
@@ -10,6 +10,18 @@
         <assemblyIdentity name="WebGrease" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.5.2.14234" newVersion="1.5.2.14234" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="JavaScriptEngineSwitcher.Core" publicKeyToken="c608b2a8cc9e4472" culture="neutral" />
```

Comment:
@jbaxleyiii We've noticed that you've added several new dependencies to these other projects that we're not certain need to be there.  Have you verified that these are needed?  We'd like to remove them if they are not needed. Less > More ;)